staging/src/k8s.io/client-go/rest/watch/decoder_test.go

```diff
@@ -70,7 +70,7 @@ func TestDecoder(t *testing.T) {
 		go func() {
 			action, got, err := decoder.Decode()
 			if err != nil {
-				t.Fatalf("Unexpected error %v", err)
+				t.Errorf("Unexpected error %v", err)
```

Comment:
Same as above.
It's interesting that this part is in a go routine at all. It doesn't seem to accomplish anything because it's done in lockstep using the done channel, but I don't think that necessarily needs to be cleaned up as part of a stacticcheck failure fix (although removing it might make this easier to fix?)

Reply:
> 与上述相同。
> 有趣的是，这部分完全在go例程中。它似乎并没有完成任何事情，因为它是通过使用done通道以同步方式完成的，但是我不认为必须将其作为stacticcheck故障修复程序的一部分进行清理（尽管将其删除可能更容易解决此问题？）

`//although removing it might make this easier to fix?`
yep.